js/main.js

```diff
@@ -209,7 +209,7 @@ function checkWin() {
     if (board.toString() == currentSolution.toString()) {
         headline.innerHTML = 'you win!';
     } else if (board.toString() !== currentSolution.toString()) {
-        headline.innerHTML = 'nonogram puzzle';
+        headline.innerHTML = 'nonogram puzzle' + ' ' + (currentPuzzle + 1);
```

Comment:
This solution works, but I wonder if we could use a variable for the current puzzle name instead. That way, if we want to change the name to something other than "nonogram puzzle", we don't have to remember to copy and paste our change to every place we update the puzzle name.